.changelog/unreleased/features/ibc-relayer-cli/2852-add-debug-flag.md

```diff
@@ -0,0 +1,3 @@
+- Add a global flag `--debug` which allows enabling
+  profiling, rpc debug logs and/or profiling to json file
```

Comment:
```suggestion
- Add a global flag `--debug` which can take one or more of the following values, separated by commas:
  - `rpc`: show RPC debug logs
  - `profiling`: show profiling information in the console
  - `profiling-json`: dump the profiling information to a JSON file in the directory specified in `PROFILING_DIR` env variable if present, or the current directory otherwise.
```